examples/local_transactions/local_transactions.bal

```diff
@@ -12,29 +13,30 @@ function main(string... args) {
         poolOptions:{maximumPoolSize:5}
     };
 
-    int updatedRows;
     //Create the tables required for the transaction.
-    var updatedRowsResult = testDBEP->update("CREATE TABLE IF NOT EXISTS CUSTOMER (ID INT, NAME VARCHAR(30))");
-    match updatedRowsResult {
-        int rows => updatedRows = rows;
+    var ret = testDBEP->update("CREATE TABLE IF NOT EXISTS CUSTOMER (ID INT, NAME VARCHAR(30))");
+    match ret {
+        int retInt => log:printInfo("CUSTOMER table create status in DB:" + retInt);
         error err => {
-            io:println("CUSTOMER table Creation failed:" + err.message);
+            handleError("CUSTOMER table Creation failed", err, testDBEP);
             return;
         }
     }
-    updatedRowsResult = testDBEP->update("CREATE TABLE IF NOT EXISTS SALARY (ID INT, MON_SALARY FLOAT)");
-    match updatedRowsResult {
-        int rows => updatedRows = rows;
+    ret = testDBEP->update("CREATE TABLE IF NOT EXISTS SALARY (ID INT, MON_SALARY FLOAT)");
+    match ret {
+        int retInt => log:printInfo("SALARY table create status in DB:" + retInt);
         error err => {
-            io:println("SALARY table Creation failed:" + err.message);
+            handleError("SALARY table Creation failed", err, testDBEP);
```

Comment:
Don't mix io:println and log statements in the sample. Let's simply use io:printlns.